packages/amplify-util-mock/src/__e2e_v2__/index-transformer.e2e.test.ts

```diff
@@ -63,9 +62,6 @@ describe('@index transformer', () => {
 
     const transformer = new GraphQLTransform({
       transformers: [new ModelTransformer(), new PrimaryKeyTransformer(), new IndexTransformer(), new AuthTransformer()],
-      featureFlags: {
-        getBoolean: name => (name === 'improvePluralization' ? true : false),
-      } as FeatureFlagProvider,
     });
```

Comment:
don't we need to set other params except `improvePluralization` to `false` here?

Reply:
In general I started by removing any flags that were here, since we're probably not testing that behavior, and only patching as tests failed. This way we're not needlessly preserving old test state.